File: leetcode/1678-goal-parser-interpretation.php
<?php

/**
 * 1678. Goal Parser Interpretation
 * https://leetcode.com/problems/goal-parser-interpretation/
 */

class Solution
{
    /**
     * @param String $command
     * @return String
     */
    function interpret($command)
    {
        return str_replace(['()', '(al)'], ['o', 'al'], $command);
    }
}

File: leetcode/1678-goal-parser-interpretation.swift
/**
 * 1678. Goal Parser Interpretation
 * https://leetcode.com/problems/goal-parser-interpretation/
 */

 class Solution {
     func interpret(_ command: String) -> String {
         return command.replacingOccurrences(of: "()", with: "o").replacingOccurrences(of: "(al)", with: "al")
     }
 }